src/pages/steps/AdditionalInformationStep/BookingInfoStep.tsx

```diff
@@ -365,30 +369,65 @@ const BookingInfoStep = ({
 
   const addBookingInfoMutation = useAddOfferBookingInfoMutation({
     onSuccess: onSuccessfulChange,
+    onMutate: async (newPayload) => {
+      await queryClient.cancelQueries({
+        queryKey: ['events', { id: eventId }],
+      });
+
+      const previousEventInfo: any = queryClient.getQueryData([
```

Comment:
```suggestion
      const previousEventInfo = queryClient.getQueryData([
```

Is the explicit any needed? Doesn't it return any as type when you don't pass a generic?